src/ostorlab/serve_app/oxo.py

```diff
@@ -505,11 +505,12 @@ def mutate(root, info: graphql_base.ResolveInfo, scan_id: int):
 
         """
         with models.Database() as session:
-            scan = session.query(models.Scan).get(scan_id)
-            if scan is None:
-                raise graphql.GraphQLError("Scan not found.")
-            local_runtime.LocalRuntime().stop(scan_id=str(scan_id))
-            return StopScanMutation(scan=scan)
+            scans = session.query(models.Scan).filter(models.Scan.id.in_(scan_ids)).all()
+            if len(scans) == 0:
+                raise graphql.GraphQLError("No scan is found.")
+            for scan_id in scan_ids:
+                local_runtime.LocalRuntime().stop(scan_id=str(scan_id))
+            return StopScanMutation(scans=scans)
```

Comment:
i think the mutation should be renamed to StopScansMutation

Reply:
+ you need to change in RE_OXO as well

Reply:
ping @oussamaessaji

Reply:
+1

Reply:
Should I update it regardless of the numerous mutations that follow this naming in RE?

Reply:
PING @oussamaessaji